basic/exercise11/script.js

```diff
@@ -7,10 +7,7 @@ function convertTemperatureFromCelsiusToFahrenheit(degreesCelsius) {
 
 function convertTemperatureFromFahrenheitToCelsius(degreesFahrenheit) {
     var degreesCelsius = (degreesFahrenheit - 32) * 5 / 9;
-    return degreesCelsius;
+    //return degreesCelsius;
+    var fixedDegreesCelsius = degreesCelsius.toFixed(2);
```

Comment:
toFixed should not be put in implementation.
function that returns value should not do rounding, because different components that call this function may have different requirements - one of them would like to have it display in 2 digits precision and other in 4 digits precision. So in general it should be made on callers side, not inside function.